stdlib/slack/slack.flux

```diff
@@ -10,26 +10,20 @@ option defaultURL = "https://slack.com/api/chat.postMessage"
 // `message` sends a single message to a Slack channel. It will work either with the chat.postMessage API or with a slack webhook.
 // `url` - string - URL of the slack endpoint. Defaults to: "https://slack.com/api/chat.postMessage", if one uses the webhook api this must be acquired as part of the slack API setup. This URL will be secret. Don't worry about secrets for the initial implementation.
 // `token` - string - the api token string.  Defaults to: "", and can be ignored if one uses the webhook api URL.
-// `username` - string - Username posting the message.
 // `channel` - string - Name of channel in which to post the message. No default.
-// `workspace` - string - Name of the slack workspace to use if there are multiple. Defaults to empty string.
 // `text` - string - The text to display.
-// `iconEmoji` - string - Name of : enclose emoji to use as image of user when posting message, will not show as the avatar icon with a slack webhook.
 // `color` - string - Color to give message: one of good, warning, and danger, or any hex rgb color value ex. #439FE0.
-message = (url=defaultURL, token="", username, channel, workspace, text, iconEmoji, color) => {
+message = (url=defaultURL, token="", channel, text, color) => {
```

Comment:
I decided to just change Slack because the pagerduty attribute that is causing errors seems to be nested within another attribute that is used in InfluxDB. We can check if these changes to slack's api work and then I'll spend time trying to figure out how to fix pagerduty.